extensions/vertx/runtime/src/main/java/io/quarkus/vertx/runtime/VertxProducer.java

```diff
@@ -37,25 +41,58 @@ public EventBus eventbus() {
 
     @Singleton
     @Produces
-    public io.vertx.axle.core.Vertx axle() {
+    public synchronized io.vertx.mutiny.core.Vertx mutiny() {
+        if (mutinyVertx == null) {
+            mutinyVertx = io.vertx.mutiny.core.Vertx.newInstance(vertx);
+        }
+        return mutinyVertx;
+    }
+
+    @Singleton
+    @Produces
+    @Deprecated
+    public synchronized io.vertx.axle.core.Vertx axle() {
+        if (axleVertx == null) {
+            LOGGER.warn(
+                    "`io.vertx.axle.core.Vertx` is deprecated - it is recommended to switch to `io.vertx.mutiny.core.Vertx`");
```

Comment:
Usually, we even say "is deprecated and will be removed in a future version". Because I suppose that's the case, isn't it?

Reply:
Yes, it's a good idea to inform the user of future removal.

Reply:
Fixed, also changed the bytecode version (related to `@ConsumeEvent`)